database-seeds/scripts/rerunners/add-table-and-folders.js

```diff
@@ -1,13 +1,12 @@
 const { MutateCollection, CreateFolderID } = require("../util");
 
 // Change these for whatever table you are adding.
-const GAME = "pms";
-const PLAYTYPES = ["Controller", "Keyboard"];
-const PREFIX = "PLv";
-const TITLE = "Normal PMS Database";
-const SHORTTITLE = "normalpmsdb"; // this is used in the tableID
-const DESCRIPTION =
-	"The Normal PMS Database. This contains almost every PMS file ever created, lv1-45.";
+const GAME = "ddr";
+const PLAYTYPES = ["SP", "DP"];
+const PREFIX = "diff";
```

Comment:
This prefix should be `"Level "`, so the folders are called `"Level 1"`, `"Level 2"`, etc.

Currently, the folder names are "diff1", "diff2", which isn't very pretty.